models/Comment.js

```diff
@@ -0,0 +1,45 @@
+/**
+ *
+ * @param {Sequelize} sequelize
+ * @param {Sequelize.DataTypes} DataTypes
+ * @returns {Sequelize.models} - Comment model
+ */
+
+const Comment = (sequelize, DataTypes) => {
+  const CommentSchema = sequelize.define(
+    'Comment',
+    {
+      body: {
+        type: DataTypes.TEXT,
+        allowNull: false,
+      },
+      user: {
```

Comment:
#### Critical
- I thought we agreed on have this renamed and refactored? @thislekan 
```suggestion
      isAnonymous: {
```

Reply:
Hello @aanchirinah .

You're currently looking at the wrong commit. I have since done as suggested. see the commit message here [commit for renaming user column](https://github.com/andela/apollo-ah-backend/pull/41/commits/f32e72a456b68694d56e4072d7833c4f3c1fc764).